src/index.ts

```diff
@@ -223,11 +228,9 @@ export default class OtaClient {
         const files = await this.getJsonFiles(file);
         const languages = await this.listLanguages();
         const res: LanguageStrings = {};
-        await Promise.all(
-            languages.map(async language => {
-                res[language] = await this.getStringsByFilesAndLocale(files, language);
-            }),
-        );
+        for (const language of languages) {
```

Comment:
and here....e.g. if we have 3 languages and `getStringsByFilesAndLocale` takes ~1 second, your code will be executed in 3 seconds and previously it could handle it within 1 second as all 3 requests were fired immediately and we are just waiting for response